tests/export-events-network-error.test.ts

```diff
@@ -1,7 +1,7 @@
 import { RetryError } from '@posthog/plugin-scaffold'
 import fetch from 'node-fetch'
 
-import { BrazeMeta, onEvent, setupPlugin } from '..'
+import { BrazeMeta, exportEvents, setupPlugin } from '..'
 
 jest.mock('node-fetch')
```

Comment:
Off topic:

As you didn't change this, but one big advantage of using something like MSW over `node-fetch` is we can be pretty sure that if we mock out with a timeout or 5xx error then the error propagation will be representative of non-mocked usage.